docs/how-to/h-async-replication.md

```diff
@@ -0,0 +1,321 @@
+                    "status": "ONLINE", 
+                    "version": "8.0.32"
+                }, 
+                "mysql-2nd-1": {
+                    "address": "10.18.215.87:3306", 
+                    "memberRole": "SECONDARY", 
+                    "mode": "R/O", 
+                    "replicationLagFromImmediateSource": "", 
+                    "replicationLagFromOriginalSource": "", 
+                    "status": "ONLINE", 
+                    "version": "8.0.32"
+                }
+            }, 
+            "transactionSet": "0a2d1861-0ea1-11ee-8ec2-00163ef59470:1-17047,0a2d1b28-0ea1-11ee-8ec2-00163ef59470:1-5,1549f330-0ea2-11ee-95a9-00163e5d863b:1-7,ec27d3f3-0ea0-11ee-aa9d-00163ef59470:1-4", 
+            "transactionSetConsistencyStatus": "OK", 
+            "transactionSetErrantGtidSet": "", 
+            "transactionSetMissingGtidSet": ""
+        }, 
+        "ROME": {
+            "clusterRole": "PRIMARY", 
+            "globalStatus": "OK", 
+            "primary": "10.18.215.67:3306", 
+            "status": "OK", 
+            "statusText": "Cluster is ONLINE and can tolerate up to ONE failure.", 
+            "topology": {
+                "mysql-1st-0": {
+                    "address": "10.18.215.9:3306", 
+                    "memberRole": "SECONDARY", 
+                    "mode": "R/O", 
+                    "replicationLagFromImmediateSource": "", 
+                    "replicationLagFromOriginalSource": "", 
+                    "status": "ONLINE", 
+                    "version": "8.0.32"
+                }, 
+                "mysql-1st-1": {
+                    "address": "10.18.215.81:3306", 
+                    "memberRole": "SECONDARY", 
+                    "mode": "R/O", 
+                    "replicationLagFromImmediateSource": "", 
+                    "replicationLagFromOriginalSource": "", 
+                    "status": "ONLINE", 
+                    "version": "8.0.32"
+                }, 
+                "mysql-1st-2": {
+                    "address": "10.18.215.67:3306", 
+                    "memberRole": "PRIMARY", 
+                    "mode": "R/W", 
+                    "status": "ONLINE", 
+                    "version": "8.0.32"
+                }
+            }, 
+            "transactionSet": "0a2d1861-0ea1-11ee-8ec2-00163ef59470:1-17047,0a2d1b28-0ea1-11ee-8ec2-00163ef59470:1-5,1549f330-0ea2-11ee-95a9-00163e5d863b:1-6,ec27d3f3-0ea0-11ee-aa9d-00163ef59470:1-4"
+        }
+    }, 
+    "domainName": "cluster-set-0a16c7e0e4d4a8c8956911b8cadb17f3", 
+    "globalPrimaryInstance": "10.18.215.67:3306", 
+    "metadataServer": "10.18.215.67:3306", 
+    "primaryCluster": "ROME", 
+    "status": "HEALTHY", 
+    "statusText": "All Clusters available."
+}
+ MySQL  127.0.0.1:33060+ ssl  Py > 
+```
+
+All OK here!
+
+## MySQL Router
+Removing test application to bootstrap two independent test applications behind mysql-router to provide the complete setup as displayed on the diagram above:
```

Comment:
What happens if we configure the router from the very beginning? Will it automatically discover new topology?

Reply:
If mysql-router bootstrap BEFORE ClusterSet created (old installations?) it have to be [rebootstrapped](https://videohub.oracle.com/media/Disaster+Recovery+with+MySQL+InnoDB+ClusterSet+-+What+is+it+and+how+do+I+use+itF/1_w0naxll1/170883371#t=27:50).
Otherwise should work, but didn't it test it, pending mysql-router refactoring.